meta-xt-prod-devel-rcar-driver-domain/recipes-graphics/recipes-graphucs/wayland/weston-init.bbappend

```diff
@@ -0,0 +1,10 @@
+do_install:append() {
+    # Turn off usage of the LVDS connector in case of virtio
+    if ${@bb.utils.contains('DISTRO_FEATURES', 'enable_virtio', 'true', 'false', d)}; then
+        sed -e '$a\\' \
```

Comment:
Is this a fancy way to write
```
echo "[output] 
name=LVDS-1
mode=off" >>  ${D}/${sysconfdir}/xdg/weston/weston.ini
```
? :)

Reply:
Can I get some answer, please?

Reply:
Hi @lorc, this, as all other minor findings here and in meta-xt-common, are accepted and will be fixed.

I'm trying to resolve your big comment which asks me to change "many huge things". Trying to find motivation for doing that first. ))